Error building the site

I cloned the repository, ran the build command given in the README, and this is what happened. I see that `ScrollingModule` is only available in `@angular/cdk-experimental`, but this repository doesn't use it or even depend on it. How could I manage to build the site successfully?
Thanks in advance.

```
npm run publish-prod

> material-angular-io@0.0.0 publish-prod /home/bryan/material.angular.io
> bash ./tools/deploy.sh stable prod

> material-angular-io@0.0.0 build-themes /home/bryan/material.angular.io
> bash ./tools/build-themes.sh

Building custom theme scss files.
Finished building CSS.

Date: 2018-08-05T21:34:44.548Z
Hash: b6116c2108ee56a9d7bb
Time: 121857ms
chunk {0} runtime.a66f828dca56eeb90e02.js (runtime) 1.06 kB [entry] [rendered]
chunk {1} purple-green.dbecf5619b9ba2c0d404.css (purple-green) 44.4 kB [initial] [rendered]
chunk {2} indigo-pink.4cc8b8a52e0e3fa474a3.css (indigo-pink) 43 kB [initial] [rendered]
chunk {3} deeppurple-amber.dad821f5ee1f4d9ad1c8.css (deeppurple-amber) 42.9 kB [initial] [rendered]
chunk {4} pink-bluegrey.33919ece7d00f01aca8c.css (pink-bluegrey) 44.4 kB [initial] [rendered]
chunk {5} styles.b966cb2e09e9d54ec83b.css (styles) 64.8 kB [initial] [rendered]
chunk {6} main.b7365267315595ed5049.js (main) 130 bytes [initial] [rendered]

ERROR in : Unexpected value 'ScrollingModule in /home/bryan/material.angular.io/node_modules/@angular/cdk/scrolling.d.ts' imported by the module 'ɵhb in /home/bryan/material.angular.io/node_modules/@angular/material-examples/typings/index.d.ts'. Please add a @NgModule annotation.

Asset-group 'app' in 'ngsw-config.json' uses the 'versionedFiles' option.
As of v6 'versionedFiles' and 'files' options have the same behavior. Use 'files' instead.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! material-angular-io@0.0.0 publish-prod: `bash ./tools/deploy.sh stable prod`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the material-angular-io@0.0.0 publish-prod script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/README.md
+++ b/README.md
@@ -1,7 +1,51 @@
 # Angular Material Docs Site
 
-## Preview
-https://material2-docs-dev.firebaseapp.com/
+This is the repository for the [Angular Material documentation site](https://material.angular.io/).
+
+Versions of this site are also available for
+- [Beta](https://next.material.angular.io/)
+- [v5](https://v5.material.angular.io/)
+- [v6](https://v6.material.angular.io/)
+
+## Contributing
+Please open bugs against the Angular Material components, directives, documentation contents, API docs,
+and demos in the [Angular Material repo](https://github.com/angular/material2/issues).
+
+Please only open issues with the documentation site itself (not the content) in
+[this repo](https://github.com/angular/material.angular.io/issues). This includes issues like the navigation
+not working properly, examples or documentation not being presented in an accessible way, issues with
+rendering or layout of the documentation pages, etc.
+
+### Where does the content come from?
+The documentation is generated from the following resources
+- [Guides](https://github.com/angular/material2/tree/master/guides)
+- [Examples](https://github.com/angular/material2/tree/master/src/material-examples)
+- [Material components, services, and directives](https://github.com/angular/material2/tree/master/src/lib)
+- [CDK components, services, and directives](https://github.com/angular/material2/tree/master/src/cdk)
+
+## Development Setup
+1. Clone Angular Material in the parent directory of this repo
+    1. `cd ..`
+    1. `git clone git@github.com:angular/material2.git`
+1. Install Gulp globally: `npm i -g gulp`
+1. Build and copy docs and examples from Angular Material: `npm run fetch-local`
+    - Note that you may need to run this after each time that you run `npm i` as some of the examples are
+      actually placed in `node_modules/@angular/`
+    - If you see the error `Cannot find module '@angular/material-examples'`,
+      it means that you need to run `npm run fetch-local` again
+
+## Development Server
+1. Run `npm start` for a dev server. Navigate to `http://localhost:4200/`
+
+## Build
+Run `npm run prod-build` to build the project.
+
+## Running unit tests
+1. Run `npm run test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+## Running end-to-end tests
+Run `npm run e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
+Before running the tests make sure you are serving the app via `npm start`.
 
 ## Deployment instructions
 ```
@@ -13,35 +57,3 @@ https://material2-docs-dev.firebaseapp.com/
 # Production
 > npm run publish-prod
 ```
-
-For development builds use the usual `ng serve` and `ng build` commands.
-
-
-----------------------------------------------------------------------------------------------------------------
-
-
-This project was generated with [angular-cli](https://github.com/angular/angular-cli) version 1.0.0-beta.19-3.
-
-## Development server
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
-
-## Code scaffolding
-
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive/pipe/service/class`.
-
-## Build
-
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
-
-## Running unit tests
-
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
-
-## Running end-to-end tests
-
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
-Before running the tests make sure you are serving the app via `ng serve`.
-
-## Further help
-
-To get more help on the `angular-cli` use `ng --help` or go check out the [Angular-CLI README](https://github.com/angular/angular-cli/blob/master/README.md).

